services/frontend-service/pkg/cmd/server.go

```diff
@@ -486,3 +487,10 @@ func (p *GrpcProxy) StreamStatus(in *api.StreamStatusRequest, stream api.Rollout
 		}
 	}
 }
+
+func (p *GrpcProxy) GetStatus(ctx context.Context, in *api.GetStatusRequest) (*api.GetStatusResponse, error) {
+	if p.RolloutServiceClient == nil {
+		return nil, status.Error(codes.Unimplemented, "rollout status not implemented")
```

Comment:
```suggestion
		return nil, status.Error(codes.Unimplemented, "rollout status not implemented, service client not configured")
```
(or something like that)